docs/_ext/import_projects.py

```diff
@@ -19,11 +19,13 @@
 class ImportProjectsConfig:
     clone_dir: str
     flytekit_api_dir: str
+    show_repo_tags: bool
```

Comment:
Can we always display the tags and commits? This way, we do not need another option to configure it.

Reply:
done